projects/project-2/assignment/Miroir SPARQL Queries/ReadME.md

```diff
@@ -0,0 +1,154 @@
+### NOTES FOR ASSIGNMENT
```

Comment:
You've mixed in project 2 with this pull request; please disentangle (when you can, obviously no rush)